tools/validators/instance_validator/validate/entity_instance.py

```diff
@@ -439,8 +506,9 @@ def _ConnectionsAreValid(self, entity: EntityInstance) -> bool:
     for conn_inst in entity.connections:
       conn_universe = self.universe.connection_universe
       if conn_universe and not conn_universe.IsDefined(conn_inst.ctype):
-        print(f'Connection type: {conn_inst.ctype} '
-              'is undefined in the ontology')
+        print(f'[ERROR]\tEntity {entity.guid} ({entity.code}) defines '
+              f'connection {conn_inst.ctype}, which does not exist in the '
+              f'ontology.')
```

Comment:
nit: no need for an `f` on the last line?

Reply:
+ 1